packages/generator/ni_measurement_plugin_sdk_generator/client/__init__.py

```diff
@@ -89,91 +290,66 @@ def create_client(
+    discovery_client = DiscoveryClient(grpc_channel_pool=channel_pool)
+    built_in_import_modules: List[str] = []
+    custom_import_modules: List[str] = []
 
-        measurement_service_stub = get_measurement_service_stub(
-            discovery_client, channel_pool, service_class
-        )
-        metadata = measurement_service_stub.GetMetadata(
-            v2_measurement_service_pb2.GetMetadataRequest()
+    while True:
+        _create_client(
+            interactive_mode=True,
+            channel_pool=channel_pool,
+            discovery_client=discovery_client,
+            built_in_import_modules=built_in_import_modules,
+            custom_import_modules=custom_import_modules,
         )
-        configuration_metadata = get_configuration_metadata_by_index(metadata, service_class)
-        output_metadata = get_output_metadata_by_index(metadata)
 
-        configuration_parameters_with_type_and_default_values, measure_api_parameters = (
-            get_configuration_parameters_with_type_and_default_values(
-                configuration_metadata, built_in_import_modules
+        selection = (
+            click.prompt(
+                "\nEnter 'Y' to continue, or enter any other keys to exit",
+                type=str,
+                default="",
+                show_default=False,
             )
+            .strip()
+            .lower()
         )
-        output_parameters_with_type = get_output_parameters_with_type(
-            output_metadata, built_in_import_modules, custom_import_modules
-        )
+        if selection == "y":
+            continue
+        else:
+            break
 
-        _create_file(
-            template_name="measurement_plugin_client.py.mako",
-            file_name=f"{module_name}.py",
-            directory_out=directory_out_path,
-            class_name=class_name,
-            display_name=metadata.measurement_details.display_name,
-            configuration_metadata=configuration_metadata,
-            output_metadata=output_metadata,
-            service_class=service_class,
-            configuration_parameters_with_type_and_default_values=configuration_parameters_with_type_and_default_values,
-            measure_api_parameters=measure_api_parameters,
-            output_parameters_with_type=output_parameters_with_type,
-            built_in_import_modules=to_ordered_set(built_in_import_modules),
-            custom_import_modules=to_ordered_set(custom_import_modules),
-        )
 
-        print(
-            f"The measurement plug-in client for the service class '{service_class}' has been created successfully."
-        )
+@click.group()
+def create_client() -> None:
+    """Generates a Python Measurement Plug-In Client module for the measurement service.
+
+    You can use the generated module to interact with the corresponding measurement service.
+
+    Use command 'b' to generate the client module in batch mode, or 'i' for interactive mode.
```

Comment:
Commands should be words, not single letters.

Also, commands are typically verbs like "generate" or "build", not adjectives like "batch" or "interactive".

Reply:
As the approach got changed. Now we don't need sub commands

Reply:
@MounikaBattu17 Please update the PR description accordingly.

Reply:
Updated.